packages/web-forms/src/components/widgets/LikertWidget.vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
+import type { SelectNode } from '@getodk/xforms-engine';
+import RadioButton from '../widgets/RadioButton.vue';
+
+defineProps<{question: SelectNode}>();
+
+</script>
+
+<template>
+	<div class="likert">
+		<RadioButton :question="question" />
+	</div>
+</template>
+
+<style scoped lang="scss">
+.likert {
+	display: flex;
+	flex-wrap: nowrap;
+	flex-direction: row;
+	line-height: 20px;
+
+	:deep(.value-option) {
+		display: flex;
+		flex-direction: column;
+		flex: 1;
+		align-items: center;
+		cursor: pointer;
+		outline: none;
+		padding:0;
+
+		&:hover {
+			background-color: unset;
+		}
+
+		&.active {
+			background-color: unset;
+			outline: none;
+		}
+
+
+		.p-radiobutton {
+			position: relative;
+			z-index: 10;
+		}
+
+		span {
```

Comment:
Is there a reason not to assign a class to target this selector?

Reply:
(Same goes for any bare tag name selector, really.)

Reply:
naming things is hard 🙃